rewrite-maven/src/main/java/org/openrewrite/maven/MavenVisitor.java

```diff
@@ -76,6 +81,13 @@ public boolean isDependencyTag() {
         return DEPENDENCY_MATCHER.matches(getCursor());
     }
 
+    public boolean isRepositoryUrlTag() {
+        return REPOSITORY_URL_MATCHER.matches(getCursor()) ||
+                PLUGIN_REPOSITORY_URL_MATCHER.matches(getCursor()) ||
+                DISTRIBUTION_MANAGEMENT_REPOSITORY_URL_MATCHER.matches(getCursor()) ||
+                DISTRIBUTION_MANAGEMENT_SNAPSHOT_REPOSITORY_URL_MATCHER.matches(getCursor());
+    }
+
```

Comment:
Would you mind moving these to the `UpdateMavenPropertyToHttpsVisitor`?  We can always move this into the `MavenVisitor` if the need for knowing repository-url tags becomes more common.